doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1131,6 +1131,9 @@ Libraries for NFC
 nRF Security
 ------------
 
+* PSA crypto is now supported on nRF54L Series devices.
```

Comment:
```suggestion
* PSA crypto using hardware acceleration is now supported on nRF54L Series devices.
```
Rationale: SW enabled with nrf_oberon was already supported

Reply:
No, 54L was not supported in 2.5.0, so PSA+54L was not supported either.